retroshare-gui/src/qss/retroclassic.qss

```diff
@@ -0,0 +1,452 @@
+/* Default stylesheet
+   This file is used as default for all stylesheets and can be overloaded
+   Don't use font-size or font-weight, they are defined in .ui files.
+   Icon size are defined in .cpp files.
+
+   For decoration, use px (pixel or dot)
+   For horizontal size, use em (width of M char)
+   For vertical size, use ex (height of x char to respect DotPerInch of your system)
+   See: https://doc.qt.io/qt-5/stylesheet-reference.html#length
+   https://doc.qt.io/qt-5/stylesheet-syntax.html
+   */
+
+/**** Standard rules ****/
+
+/* Transparent background objects = name starting by trans_*/
+QObject[objectName^="trans_"] {
+	background: transparent;
+}
+/* Specific for scrollarea contents */
+QScrollArea[objectName^="trans_"] > QWidget > QWidget {
+	background: transparent;
+}
+
+/* Informatives background objects = name starting by info*/
+QObject[objectName^="info_"]:enabled {/*enabled is needed for not be overrided :https://doc.qt.io/qt-5/stylesheet-syntax.html#conflict-resolution*/
+	color: black;
+	border: 1px solid #DCDC41;
+	border-radius: 6px;
+	background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #FFFFD7, stop:1 #FFFFB2);
+}
+/* For label inside informative frame */
+QFrame[objectName^="info_"] > QLabel
+{
+	color: black;
+	background: transparent;
+}
+
+/* HeaderFrame & TitleBarFrame */
+
+QFrame[objectName^="headerFrame"] {
+	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #9BDBF9, stop:1 #1592CD);
+}
+QFrame[objectName^="headerBFrame"] {
+	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #9BDBF9, stop:1 #1592CD);
+	border: 1px;
+	border-radius: 4px;
+}
+QFrame[objectName^="headerFrame"] > *:!hover,
+QFrame[objectName^="headerBFrame"] > *:!hover{
+	background: transparent;
+	color: white;
+}
+
+QFrame[objectName^="toolBarFrame"],
+QFrame[objectName^="toolBarFrameTop"] {
+	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #FEFEFE, stop:1 #E8E8E8);
+	border: 1px solid #CCCCCC;
+}
```

Comment:
Only these lines are needed as all other are in default.qss.

Reply:
As I've already wrote, default.qss is load every time (that's its name). When you choose another in list, RS loads it after default.qss. So you can override some properties.
The code is here:
https://github.com/RetroShare/RetroShare/blob/master/retroshare-gui/src/rshare.cpp#L690

Reply:
ok i now i understand it, i need do some tests